specs/simple-serialize.md

```diff
@@ -177,38 +175,36 @@ Note that deserialization requires hardening against invalid inputs. A non-exhau
 
 We first define helper functions:
 
+* `chunk_count(type)`: calculate the amount of leafs for merkleization of the type.
+   * all basic types: `1`
+   * bitlists and bitvectors: `(N + 255) // 256` (dividing by chunk size, rounding up)
```

Comment:
`chunk_count` only takes `type` as input so `N` ends up being kind of implicitly available. I suppose that's okay, thoughts?

Reply:
replaced type names with type descriptions, to source the `N` and element type from. See d9fd1d3